docs/_articles/en/soql/soql-tasks.md

```diff
@@ -0,0 +1,31 @@
+---
+title: SOQL Tasks
+lang: en
+---
+
+This is a repository of sample tasks using SOQL that you can complete in Code Builder or VS Code Extensions for desktop. We’ll keep adding to this list, so make sure you visit this topic often. 
+
+
+## Use the SOQL Query Editor Create a Query To List Accounts with a New York State Address
+
+1. From the command palette, run **SFDX: Create Query in SOQL Builder** to open SOQL Query Builder.
+2. Click **File > Save** to save the query. Make sure to retain the .soql file extension.
```

Comment:
Question: they haven't actually created a query yet, so what exactly are they saving here?

Reply:
good point. changed to "file".